src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $main-card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: scale 0.3s;
+
+  &:hover {
+    scale: 1.2;
```

Comment:
The `scale` property is not a valid CSS property. You should use the `transform` property with `scale` as its value, like `transform: scale(1.2);`.